plane/src/client/mod.rs

```diff
@@ -138,6 +137,19 @@ impl PlaneClient {
         Ok(())
     }
 
+    pub async fn revoke_token(
+        &self,
+        backend_id: &BackendName,
+        username: &str,
```

Comment:
Ah good catch, went with the second option to move `backend_id` and `user` into a JSON payload because it feels more consistent given we handle `user` in JSON in `connect`. 

I took the opportunity to clean up the code a bit and leverage typing more so we don't have to redo a bunch of function signatures should we want to change the arguments to the API (like allowing folks to use `key` etc.). 

The API is now `/ctrl/revoke` and renamed a bunch of functions accordingly. Lmk if you prefer a different naming like `revoke-token` though.